src/applications/simple-forms/20-10207/helpers.js

```diff
@@ -122,3 +122,27 @@ export function dateOfDeathValidation(errors, fields) {
     );
   }
 }
+
+export function powConfinementDateRangeValidation(errors, fields) {
```

Comment:
This was another one I was thinking about patternizing, as date ranges are pretty common. like maybe just provide the validation as a pattern where you can specify the two fields.

Reply:
Yeah, definitely would be useful to have a range-pattern with range-validation.